Marlin/src/lcd/ultralcd.cpp

```diff
@@ -4577,7 +4577,7 @@ void lcd_quick_feedback(const bool clear_buttons) {
       _change_filament_temp_extruder = extruder;
       START_MENU();
       if (LCD_HEIGHT >= 4) STATIC_ITEM_P(change_filament_header(mode), true, true);
-      MENU_BACK(MSG_FILAMENTCHANGE);
+      MENU_BACK(MSG_MAIN);
```

Comment:
This submenu is listed in both `lcd_change_filament_menu` and `lcd_main_menu`. This should be changed to `MSG_BACK` if you're trying to generalize.